openmc/capi/math.py

```diff
@@ -0,0 +1,273 @@
+from ctypes import (c_int, c_double, POINTER)
+
+import numpy as np
+from numpy.ctypeslib import ndpointer
+
+from . import _dll
+
+_dll.t_percentile.restype = c_double
+_dll.t_percentile.argtypes = [POINTER(c_double), POINTER(c_int)]
+
+_dll.calc_pn.restype = None
+_dll.calc_pn.argtypes = [POINTER(c_int), POINTER(c_double),
+                         ndpointer(c_double)]
+
+_dll.evaluate_legendre.restype = c_double
+_dll.evaluate_legendre.argtypes = [POINTER(c_int), POINTER(c_double),
```

Comment:
These changes provide a seg fault.  (One would think setting the `value` attribute on the interface in Fortran would lead to not needing these POINTERS, but for some reason I still do need them.  Haven't figured out why.

Reply:
... embarassing.  I had thought since it wasnt in the directory of attributes of _dll that I had to expose it.  But it makes sense that the calls to `restypes` and `argtypes` can also initialize the interface under the hood.